Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/JoinSessionMessage.java

```diff
@@ -26,6 +33,12 @@ public JoinSessionMessage(Status status) {
         settings = null;
     }
 
+    /**
+     * Intializes the JoinSessionMessage class.
+     * 
+     * @param roomName The given room name, not null.
+     * @param password The given password, not null.
+     */
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)